src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,11 @@
-export const Tabs = () => {};
+export const Tabs = ({ selected, tabs, sendTitle }) => {
```

Comment:
it's better rename Component since we have not tabs here but TabContent
```suggestion
export const TabContent = ({ selectedTab }) => {
```